### DISCUSSION TITLE
Not working as it should

### DISCUSSION THREAD

**Discussion by dltnc** [directus-labs/agency-os#41] [General]
The backend template import stuck with the message "Loaded Schema
Applying template - AgencyOS... \
Loaded Roles
Loaded Folders
Applying template - AgencyOS... !
    TypeError: Cannot read properties of undefined (reading 'data')"

And the front-end not working. Failed at building process. Doesn't work on vercel or netlify.

**Comment by bryantgillespie** [comment]
Do you have some more context? 

Very hard to help pinpoint any issues without the specific errors you're receiving.

Are you using Directus Cloud? Are you deploying Directus somewhere else?

Running a build on Vercel will fail unless you have a Directus instance that's accessible during the build process.

**Reply by dltnc** [reply]
Thanks for your reply. I am deploying on a CapRover image installed on google cloud instance. The backend is working fine. Still when I try to apply template, I get an error message "TypeError: Cannot read properties of undefined (reading 'data')" after below steps: 
Loaded Schema
Applying template - AgencyOS...
Loaded Roles
Loaded Folders
Applying template - AgencyOS... !

The frontend also not working. It stops while build process. I tired on local environment and vercel also.

**Reply by bryantgillespie** [reply]
I'm not familiar with CapRover.

- What are the specifics of your Directus deployment? 
- Did you use Docker and the sample compose file included?
- If not...
- What database vendor are you using? We recommend Postgres for this template because that's one we tested against.
- What version of Directus are you using?

If the Directus instance is not properly set up or the template is not correctly applied to that instance, which is what it looks like here - then obviously the build on the front end will fail because it is dependent on the Directus instance.

**Comment by dltnc** [comment]
Thanks for reply. I fixed the issue. The Websocket was disabled. After enabling, all issues are okay now.
